mobile/native/src/db/models.rs

```diff
@@ -108,16 +107,16 @@ impl Order {
         }
     }
 
-    pub fn get(order_id: Uuid, conn: &mut SqliteConnection) -> QueryResult<Order> {
+    pub fn get(order_id: String, conn: &mut SqliteConnection) -> QueryResult<Order> {
```

Comment:
I remember why I left this here as a Uuid: because I failed to store the Uuid in the DB and wanted to pick it up again. :D I think a string here is fine though .